src/runtime/crt/host/crt_config.h

```diff
@@ -46,11 +46,14 @@
 #define TVM_CRT_GLOBAL_FUNC_REGISTRY_SIZE_BYTES 256
 
 /*! Maximum packet size, in bytes, including the length header. */
-#define TVM_CRT_MAX_PACKET_SIZE_BYTES 64000
+#define TVM_CRT_MAX_PACKET_SIZE_BYTES 8 * 1024
 
 /*! \brief Maximum length of a PackedFunc function name. */
 #define TVM_CRT_MAX_FUNCTION_NAME_LENGTH_BYTES 30
 
+/*! Size of the global function for max RPC transfer, in bytes. */
+#define TVM_CRT_RPC_MAX_TRANSFER_SIZE_BYTES 2048
```

Comment:
I think the RPC function should return an RPC-level thing rather than the limit on one RPC call in particular. So I'd vote to have the function return TVM_CRT_MAX_PACKET_SIZE_BYTES, then in CopyToRemote, compute the max size of tensor that fits in TVM_CRT_MAX_PACKET_SIZE_BYTES. I think that approach should avoid needing to add this constant here.